core/sdk-core/src/main/java/software/amazon/awssdk/core/async/SdkPublisher.java

```diff
@@ -51,6 +52,10 @@ static <T> SdkPublisher<T> adapt(Publisher<T> toAdapt) {
         return toAdapt::subscribe;
     }
 
+    static <T> SdkPublisher<T> fromIterable(Iterable<T> iterable) {
```

Comment:
Ignore this file as this change is reviewed in https://github.com/aws/aws-sdk-java-v2/pull/5030 This change will be removed.